qiskit_ionq/ionq_client.py

```diff
@@ -244,7 +246,12 @@ def get_results(
             params.update(extra_query_params)
 
         req_path = self.make_path("jobs", job_id, "results")
-        res = requests.get(req_path, params, headers=self.api_headers, timeout=30)
```

Comment:
Can we also add to other requests.get calls in this file?

Ty!

Reply:
didnt mean to delete that haha nice catch